### PR TITLE
Embeddium X JVM Update

### PR DESCRIPTION
**Pull request by Cartrigger**
https://github.com/QuestCraftPlusPlus/Pojlib/pull/44
JVM Args are listed here

This PR also adds Embeddium to .1 and .4 of 1.20's, with adding Noisium to all 20s and 
This also adds all prior mods to 1.20.4, which was in need.
Also removes modmenu because that is not meant to be there by default, only in devmods by default, but because most people are using devmods now i have to treat both the same.

### CODE REVIEW

**Comment by Cartrigger** [comment]
Looks good on my end, all mods added do not have dependencies (other than fabricapi)

**Comment by computer-catt** [comment]
colon3

File: src/main/java/pojlib/util/JREUtils.java [hunk 1]
```diff
@@ -197,7 +197,10 @@ public static int launchJavaVM(final Activity activity, final List<String> JVMAr
 
         userArgs.add("-XX:+UseZGC");
         userArgs.add("-XX:+ZGenerational");
-
+        userArgs.add("-XX:+UnlockExperimentalVMOptions");
+        userArgs.add("-XX:+UnlockDiagnosticVMOptions");
```
Comment (thejudge156) [inline src/main/java/pojlib/util/JREUtils.java:201]: Is this a debug-only option or is it actually used for perf?

**Review by thejudge156 — APPROVED** [review]
I had a question, but otherwise looks good.

File: src/main/java/pojlib/util/JREUtils.java (hunk 1 above)
Comment (Cartrigger) [reply src/main/java/pojlib/util/JREUtils.java:201]: its used for perf, its to add jvm options

Comment (Cartrigger) [reply src/main/java/pojlib/util/JREUtils.java:201]: and overall perf

**Pull request merged**